lib/jsrt/JsrtSourceHolder.cpp

```diff
@@ -49,7 +51,6 @@ namespace Js
             *utf8Script = HeapNewArray(utf8char_t, cbUtf8Buffer);
         }
 
-        Assert(length < MAXLONG);
```

Comment:
> However, if the if statement condition you introduce properly guards the bound, then there's no need for the assert as it would be redundant (it's simple enough in this case).

Agree.